hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieArrayWritableAvroUtils.java

```diff
@@ -37,12 +38,17 @@ public class HoodieArrayWritableAvroUtils {
   private static final Cache<String, ObjectInspectorCache>
       OBJECT_INSPECTOR_TABLE_CACHE = Caffeine.newBuilder().maximumSize(1000).build();
 
-  public static ObjectInspectorCache getCacheForTable(String table, Schema tableSchema, JobConf jobConf) {
```

Comment:
We weren't actually storing anything in OBJECT_INSPECTOR_TABLE_CACHE. Making this work as intended might have some issues because what if there are two tables with the same name but different locations? Also schema evolution might be an issue. We probably don't want to persist the cache between queries, just in a single query we don't want to recompute each time. Or maybe the overhead of a global cache is more than the computation? How can we find this out?